pica/cli/cccv.py

```diff
@@ -8,6 +8,7 @@
 
 click.option = partial(click.option, show_default=True)
 
+
 @click.group()
 def cccv():
```

Comment:
When I run `trex -h`, the description of CCCV is truncated like "Perform nested, stratified crossvalidation over a range of..." with the rest missing.